locales/hr.json

```diff
@@ -117,17 +106,10 @@
         "action": "Poslati email poruku za ponovno postavljanje lozinke"
     },
     "reset_password_form": {
-        "title": "Resetiranje lozinke",
```

Comment:
The structure here should be kept like this (with the correct translations, of course):
  ```
"reset_password_form": {
        "title": "Reset your password",
        "password": "New password",
        "confirmation_password": "Re-enter your new password",
        "action": "Reset password"
    },
    "change_password_form": {
        "title": "Change your password",
        "current_password": "Current password",
        "password": "New password",
        "confirmation_password": "Re-enter your new password",
        "action": "Change password"
    },
```